src/components/Nav/Nav.css

```diff
@@ -0,0 +1,23 @@
+nav {

```

Comment:
On smaller screens, the navbar is quite large and the "Book swap" text is cut off.
![image](https://github.com/iO-Academy/23-sept-sarcasticfringeheads-bookswap-fe/assets/9077806/96320303-2017-46a2-a279-83bb100ac1c6)